src/NLog/Common/InternalLogger.cs

```diff
@@ -79,6 +77,7 @@ public static void Reset()
             LogToConsoleError = GetSetting("nlog.internalLogToConsoleError", "NLOG_INTERNAL_LOG_TO_CONSOLE_ERROR", false);
             LogLevel = GetSetting("nlog.internalLogLevel", "NLOG_INTERNAL_LOG_LEVEL", LogLevel.Info);
             LogFile = GetSetting("nlog.internalLogFile", "NLOG_INTERNAL_LOG_FILE", string.Empty);
+            WriteToDiagnostics = GetSetting("nlog.internalWriteToDiagnostics", "NLOG_INTERNAL_WRITE_TO_DIAGNOSTICS", false);
```

Comment:
I think this should be LogToDiagnostics (also in xml config etc), to be consistent.  Do you agree?